docs/example.ts

```diff
@@ -91,26 +98,27 @@ const mnemonicForAttester = Kilt.Identity.generateMnemonic()
 const attester = Kilt.Identity.buildFromMnemonic(mnemonicForAttester)
 
 // And we need to build a request for an attestation
-const requestForAttestation = new Kilt.RequestForAttestation(claim, [], claimer)
+const requestForAttestation = Kilt.RequestForAttestation.fromClaimAndIdentity(
+  claim,
+  claimer
+)
 
 // Excourse to the messaging system
 // If the Attester doesn't live on the same machine, we need to send her a message
 const messageBody: IRequestAttestationForClaim = {
-  content: requestForAttestation,
+  content: {
+    requestForAttestation,
+  },
   type: MessageBodyType.REQUEST_ATTESTATION_FOR_CLAIM,
 }
 const message = new Kilt.Message(messageBody, claimer, attester)
 // The message can be encrypted as follows
 const encrypted = message.getEncryptedMessage()
 
 // Check the validity of the message
-Message.ensureHashAndSignature(encrypted, claimer)
+Message.ensureHashAndSignature(encrypted, claimer.address)
 // When the Attester receives the message, she can decrypt it
-const decrypted = Message.createFromEncryptedMessage(
-  encrypted,
-  claimer,
-  attester
-)
+const decrypted = Message.createFromEncryptedMessage(encrypted, attester)
 
 // And make sure, that the sender is the owner of the identity
```

Comment:
I am not sure about this line here. What does it relate to? Why do we need to know the owner of the identity is also the sender?

Reply:
we need to make sure that the sender is the owner of the attested claim. If Alice shares her attested claim with Mallory, Mallory could send it to a verifier. If the verifier doesn't verify that sender == owner he will not notice that the attested claim belongs to Alice and not to Mallory.